### PR TITLE
Edit Exporting code sample to match updated Dodge the Creeps code

### PR DESCRIPTION
**Pull request by Ursus-Z**
Some minor edits to the Step by Step/Exporting sample code so that it matches changes made to the 3.1 Dodge the Creeps code.

### CODE REVIEW

**Comment by cbscribe** [comment]
Thanks!

File: getting_started/step_by_step/exporting.rst [hunk 1]
```diff
@@ -115,9 +115,9 @@ changed:
             $AnimatedSprite.flip_v = velocity.y > 0
 
     func _on_Player_body_entered( body ):
-        $Collision.disabled = true
         hide()
         emit_signal("hit")
+        $CollisionShape2D.call_deferred("set_disabled", true)
```
Comment (akien-mga) [inline getting_started/step_by_step/exporting.rst]: It would be better to use `set_deferred("disabled", true)`. But that can be for another PR as I guess the other tutorial needs to be changed too.

**Comment by akien-mga** [comment]
Cherry-picked for 3.1.
